File: C#/1013 - O Maior.cs
using System;

namespace Maior{
    class Program{
        static void Main(string[] args){
            int a, b, c, maiorAB, maiorABC;
            
            String linha = Console.ReadLine();
            string[] valor = linha.Split(' ');
            a = Convert.ToInt32(valor[0]); 
            b = Convert.ToInt32(valor[1]); 
            c = Convert.ToInt32(valor[2]); 
            
            maiorAB = (a + b + Math.Abs(a - b)) / 2;
            maiorABC = (maiorAB + c + Math.Abs(maiorAB - c)) / 2;
            
            Console.WriteLine(maiorABC + " eh o maior");
        }
    }
}

File: Java/1013 - O Maior.java
import java.io.IOException;
import java.util.*;

public class Main{
    public static void main(String[] args){
        int a, b, c, maiorAB, maiorABC;
        Scanner valor = new Scanner(System.in);
        
        a = valor.nextInt();
        b = valor.nextInt();
        c = valor.nextInt();
        
        maiorAB = (a + b + Math.abs(a -b ))/2;
        
        maiorABC = (maiorAB + c + Math.abs(maiorAB - c))/2;
        
        System.out.println(maiorABC + " eh o maior");
        
    }
}
